### ISSUE TITLE
Add Polish translation

### ISSUE THREAD

**Issue by johnjohndoe** [EventFahrplan/EventFahrplan#173]
It would be nice to have a Polish translation of the app. :poland: 

The _Translation Editor_ in _Android Studio_ let's you easily identify the missing translations as shown in the screenshot.

![Android-Studio-Translation-Editor](https://user-images.githubusercontent.com/144518/65724092-fe307d00-e0af-11e9-8dd6-743d6d00237b.png)

Please use the automatic formatting for the string files which Android Studio provides.

- [x] Add new translations in `app/src/main/res/values-pl`
  - Adding a new language can be done via the _Translation Editor_: the :earth_asia: earth icon at top left
  - Take `app/src/main/res/values` (English) as a reference.
  - Ignore strings with the `translatable="false"` attribute and `placeholder` strings.
  - Files to cover: `preferences.xml`, `strings.xml`
  - Adapt the date format in `app_hardcoded_subtitle` to the country.
  - Do not modify, commit files outside of the `-pl` folder.
  - Ensure that [`plurals`](https://developer.android.com/guide/topics/resources/string-resource#Plurals) are covered (`schedule_changes_dialog_number_of_sessions`, `schedule_changes_dialog_being`, `notifications_schedule_changes`). The _Translation Editor_ might not point them out.
  - Check that your translation string is not wrapped in double quotes (`"`) or includes unneeded whitespace.
- [x] Optional: Add new translations in `app/src/rc3/res/values-pl` (becomes visible when you change the _Build Variant_)
- [x] Optional: Add new translations in `app/src/ccc36c3/res/values-pl` (becomes visible when you change the _Build Variant_)
- [x] Optional: Add new translations in `app/src/cccamp2019/res/values-pl` (becomes visible when you change the _Build Variant_)

# Related
- [Localize the UI with Translations Editor](https://developer.android.com/studio/write/translations-editor.html)
- Please read the [contribution guide](https://github.com/EventFahrplan/EventFahrplan/blob/master/CONTRIBUTING.md) before you start working.

---

:left_speech_bubble: If you want to work on this issue please leave a "I like to work on this" comment here.

**Comment by kazigk** [comment]
I'll do it :)

**Comment by johnjohndoe** [comment]
@kazigk Go ahead. You are welcome.

**Comment by johnjohndoe** [comment]
Please mind that I will **not** be able to reply quickly since I am on vacation for a while (Oct 3-31).

**Comment by johnjohndoe** [comment]
:spiral_notepad: This issue is open for contribution.

**Comment by rotsq** [comment]
I like to work on this

**Comment by johnjohndoe** [comment]
@rotsq Nice. Please let me know if you need help at some point.

**Comment by rotsq** [comment]
@johnjohndoe what is the context of this message: _No fitting app installed._ (share_error_activity_not_found)?

**Comment by rotsq** [comment]
@johnjohndoe forget the last one, I managed to get the context of `share_error_activity_not_found` from the German text. 

However, I have yet another question: is there anything to be actually translated in `preferences.xml`? My feeling is that the keys partially overlap when compared to `strings.xml`. Or, let me rephrase: assuming that I translated `strings.xml` (I need to take care of plurals only), what should I do next?

**Comment by johnjohndoe** [comment]
Yes, the strings in _preferences.xml_ has to be translated, too. It should not contain duplicate keys with regards to _strings.xml_. Please exclude strings which are marked as non-translatable as pointed out above. You can take the German translation as a blueprint.

**Comment by rotsq** [comment]
OK, so apparently, when I used `strings.xml` from the `values` folder (English strings) and translated it all to Polish, the German `preferences.xml` has only the strings that are already contained in English `strings.xml`. So I used the Italian empty `preferences.xml` as a source for this file.

Last question (I think): are the three optional tasks above still valid (i.e. translating small event-related XMLs from 2019/2020)? If so, it is not a problem, I can translate them, but I prefer to ask to not to generate files that are not needed anymore.

@johnjohndoe before I'll do a pull request, you can have a look [here](https://github.com/rotsq/EventFahrplan/tree/translation-pl/app/src/main/res/values-pl) and comment in this thread if something is wrong.

**Comment by johnjohndoe** [comment]
Yes, the translation are put into `strings.xml` by Android Studio by default. It is not very comfortable. From what I know you need to move the strings for `preferences.xml` manually into the file. Sorry for the inconvenience! I can also do that - don't worry.

The event related XMLs would be appreciated since I can reuse parts for a follow-up event. Specially, the `rC3` strings will be used soon.

**Comment by rotsq** [comment]
OK. I think we are good with everything, including past events, quotes, and newlines, plus correct locations of `preference*` strings. Pull request on the way.
